projects/commudle-admin/src/app/feature-modules/sys-admin/components/admin-badges/admin-badges-form/admin-badges-form.component.html

```diff
@@ -0,0 +1,36 @@
+<nb-card class="admin-badges-form" >
+    <nb-card-header class="d-flex justify-content-space-between align-items-center">
+        Badges
+        <button [routerLink]="['/sys-admin', 'badges']" ghost nbButton shape="round" size="small">Show All Badges</button>
+    </nb-card-header>
+    <nb-card-body>
+        <form (submit)="submitForm()" [formGroup]="badgeForm" class="form-fields d-flex flex-column" >
+            <label>
+                Name*
+                <input formControlName="name" fullWidth nbInput type="text" />
+            </label>
+            <label>
+                Badge Type*
+                <nb-select formControlName="badge_type" fullWidth placeholder="Select Badge Type">
+                    <nb-option value="expert">Expert</nb-option>
+                    <nb-option value="author">Author</nb-option>
+                </nb-select>
+            </label>
+            <label>
+                Image*
+                <input #inputImage (change)="addImage($event)" accept="image/svg+xml" fullWidth nbInput type="file" />
+            </label>
+            <nb-list class="image-list">
+                <nb-list-item *ngIf="imageUploaded" >
+                  <div *ngIf="imageSrc" >
+                    <img [src]="imageSrc | safeUrl" alt="No preview available" height="200px"/>
```

Comment:
Let's not assign height here and also change alt to have the name of the file

Reply:
done